client/roundCtrl.ts

```diff
@@ -43,115 +38,73 @@ interface MsgMoreTime {
 }
 
 interface MsgDrawOffer {

```

Comment:
These "Msg" types should be in a separate file. Maybe in types.ts or a new file roundType.ts

Reply:
Done.